src/components/Column/Column.tsx

```diff
@@ -148,12 +156,18 @@ class Column extends React.Component<PropsFromRedux & ColumnProps> {
   };
 
   handleAddTask = (columnId: string) => {
-    console.log("Add Task Clicked", columnId);
     const currentTaskIds = Object.keys(this.props.allTasks.tasks);
-    const lastTaskId = currentTaskIds[currentTaskIds.length - 1];
-    const lastTaskIdNum = parseInt(lastTaskId.charAt(lastTaskId.length - 1));
-    const newTaskId = `task-${lastTaskIdNum + 1}`;
-    const newTaskIdArray = this.props.allTasks.columns[columnId].taskIds;
+    let newTaskId;
+    if (currentTaskIds.length === 0) {
+      newTaskId = "task-1";
+    } else {
+      const lastTaskId = currentTaskIds[currentTaskIds.length - 1];
+      const regex = /^\D+/g; // find all leading non-digits
+      // replace all leading non-digits with empty string
+      const lastTaskIdNum = parseInt(lastTaskId.replace(regex, ""));
```

Comment:
Does your id need to have "task-" in it? If not, you can just use the number as the id, and then don't have to use regex. 

Another way to avoid using regex would be `lastTaskId.split('-')[1]` - split the id on the `-` character, and take the piece to the right of the dash (the number).

Reply:
I like to keep the task- for the task id to separate it from column id.

I updated the code to use split instead of regex. Thanks!!!